src/components/Navbar/Navbar.style.tsx

```diff
@@ -11,7 +11,7 @@ type NavbarStyleProps = {
 }
 
 export const StyledSearchbar = styled(Searchbar)`
-  transition: max-width 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
+  transition: max-width 0.6s ${transitions.easing};
```

Comment:
I'm not sure we should use the global timing here. I wanted to make sure all the loading transitions share the same timing. This isn't related to loading - it happens to share the same value. I can make that use `timings` as well but then we should add a separate value - just for this component, so I don't think that's worth it

Reply:
I would argue it would still be good to have duration for this in the theme, in case we want to chain animations in the future or we just want to keep them consistent through the app, I think something like `fast`, `slow`, `regular` would do

Reply:
I'd say we shouldn't pick those arbitrary values at this time as it doesn't bring us any value. If we happen to need to keep some timings consistent let's put them here at that time with a proper naming

Reply:
There is little value now because we have few unsophisticated animations, but I can see several benefits:
- Lots of time saved during refactoring
- Prevention of a lot of subtle inconsistencies which would be hard to catch, like one animation being slower of faster than another one 
This will get more annoying to do as the number of animations and hardcoded values grow so I'm for doing it now that the hassle is low

Reply:
Okay, added `regular`